haunt-roll-fail/arcs/game-common.scala

```diff
@@ -820,7 +820,10 @@ object CommonExpansion extends Expansion {
                 l3./(x => Image("raid-die-" + (Raid.die.values.indexed.%(_ == x).indices.shuffle(0) + 1), styles.token))
             )
 
-            val next = BattleProcessAction(f, r, e, (l1 ++ l2 ++ l3).flatten, then)
+            val mp = (e.lores.has(MirrorPlating) && l2.num > 0).$(Intersept)
+            val sb = (f.lores.has(SignalBreaker) && f.at(r).ships == f.at(r).diff(f.damaged)).$(Intersept)
+
+            val next = BattleProcessAction(f, r, e, (l1 ++ l2 ++ l3).flatten.diff(sb) ++ mp, then)
```

Comment:
I think Signal Breaker should be able to ignore the *Intercept* from Mirror Plating too.
Also, both effects would be lost if the player uses Skirmishers.

Reply:
BattleProcessAction could be a better place to deal with both effects.

Reply:
> I think Signal Breaker should be able to ignore the *Intercept* from Mirror Plating too.

I initially thought the same thing, then saw this: https://cards.ledergames.com/faq?locale=en-US&productId=arcs#ARCS-L06

Reply:
> BattleProcessAction could be a better place to deal with both effects.

Mirror Plating requires knowledge of assault dice rolled, I suppose it can be a flag for BattleProcessAction.

Reply:
Right. Probably we are flattening dice results too early.
Let me fix that.